lms/templates/pdf_viewer.html

```diff
@@ -44,12 +44,13 @@
     <script type="text/javascript">
         PDFJS.imageResourcesPath = "${static.url('css/vendor/pdfjs/images/') | n, js_escaped_string}";
         PDFJS.workerSrc = "${static.url('js/vendor/pdfjs/pdf.worker.js') | n, js_escaped_string}";
+        PDFJS.disableWorker = true;
```

Comment:
what this specifically mean? how this resolves the problem for Mozilla Firefox browser?

Reply:
> what this specifically mean? how this resolves the problem for Mozilla Firefox browser?

There is a browser compatibility issue with PDFJS while using it with CDNs there are other ways around, but it seems disabling worker is the simplest way to counter that problem.
@DawoudSheraz @awaisdar001  

For more context, you can look into this thread.
https://github.com/mozilla/pdf.js/issues/5490#issuecomment-63322602